test/unit/client-daemonset.bats

```diff
@@ -72,6 +72,21 @@ load _helpers
   [ "${actual}" = "null" ]
 }
 
+#--------------------------------------------------------------------
+# retry-join
+
+@test "client/DaemonSet: retry join gets populated" {
+  cd `chart_dir`
+  local actual=$(helm template \
+      -x templates/client-daemonset.yaml  \
+      --set 'server.replicas=3' \
+      . | tee /dev/stderr |
+      yq -r '.spec.template.spec.containers[0].command | any(contains("-retry-join"))' | tee /dev/stderr)
```

Comment:
Not sure where my comment went! I asked if it was possible to match exactly on the retry-join but now I realize it's not easy because it's just a string at that point. Nevermind!